skycultures/greek_farnese/description.en.utf8

```diff
@@ -0,0 +1,337 @@
+<h1>The Globe of the Atlas Farnese Statue </h1>
+<p>This ancient Greek
+celestial culture is based on the marble globe of the Atlas statue in the Villa Farnese (Naples). The statue dates to the 2nd century AD, i.e. Roman times. However, it is a copy of an older Greek model, which itself has not survived. </p>
+<p >
+ The marble globe only depicts figures and no stars. The globe has a diameter of about 65 cm. The figures were not aligned according to exact point coordinates, but in a grid: the ecliptic, the celestial equator, the tropics and the polar circles provide orientation in terms of position in "latitude circles" on the celestial sphere, while the coluric circles provide a four-way division in "longitude circles". Ancient globe makers used this very coarse-meshed grid to adjust the figure drawings. 
+ </p>
+ <p>
+ The images of the globe therefore clearly show that the stonemason who created the globe had the popular doctrinal poem by Aratos of Soloi (~300 BCE) in mind. There is no reason for the claim - often inflated by popular science - that this marble globe was carved according to Hipparchus' star catalogue. This thesis has been repeatedly raised over the past ~200 years, but regularly dismissed: Both mathematical arguments and artistic representations speak against it.
+ </p>
+<p > <img width="800" src="Atlas_(Farnese_Globe).jpg"/></p>
```

Comment:
An absolute with of 800 may be bad on small monitors. Maybe you can specify as "98%", then it may even grow. (untested!)